cdc/sinkv2/eventsink/txn/txn_sink.go

```diff
@@ -14,24 +14,197 @@
 package txn
 
 import (
+	"encoding/binary"
+	"hash/crc64"
+	"sync"
+
+	"github.com/pingcap/log"
 	"github.com/pingcap/tiflow/cdc/model"
 	"github.com/pingcap/tiflow/cdc/sinkv2/eventsink"
+	"github.com/pingcap/tiflow/pkg/causality"
+	"go.uber.org/zap"
+)
+
+const (
+	conflictDetectorSlots int64 = 1024 * 1024
```

Comment:
Is it better if we offer a hidden config parameter so that we can tuning the performance on released versions?